app/views/part/edit.scala.html

```diff
@@ -85,7 +85,7 @@
                     <div class="col-sm-5">
                         @helper.inputText(form("partId"),
                             'class -> "form-control validate[required]",
-                            Symbol("data-orig-value") -> form("partId").value)
+                            Symbol("data-orig-value") -> form("partId").value.getOrElse(""))
```

Comment:
This is because we are actually no longer in Twirl output-land. (the `@helper.inputText...` puts us into standard Scala world)